docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -16,7 +16,8 @@ See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNo
 
 .. tab:: Operating Systems
 
-  macOS 10.15, 11, 12, and 13 versions with the x86 architecture, 64 bits
+  macOS 10.15, 11, 12, and 13, x86, 64-bit
+  macOS 10.15, 11, 12, and 13, ARM, 64-bit
```

Comment:
I suppose this line with OS requirements contradicts note above (L11, `> **NOTE**: The Intel® Distribution of OpenVINO™ toolkit is supported on macOS version 10.15 with Intel® processor-based machines.`)
Should this note also be updated or removed?

Reply:
Thanks for pointing it out. I noticed it too. Removed the specific mac OS version in the note. :)